core/embed/prodtest/main.c

```diff
@@ -364,6 +393,124 @@ static void test_touch(const char *args) {
   touch_deinit();
 }
 
+static void test_touch_custom(const char *args) {
+#define NUM_PARAMS 5
```

Comment:
How about using `static const int num_params = 5;` instead of #define/#undef? If think it could work better here.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/f6fb2a667cbb9ad5b9a58408813623ce9cd61dee

(used `expected_params` as `num_params` is already used for real number of provided params)